### ISSUE TITLE
not linting javascript code properly

### ISSUE THREAD

**Issue by vajahath** [eslint/eslint#8423]
**Tell us about your environment**
Ubuntu 14.04 LTS 64 bit

* **ESLint Version:** v3.19.0
* **Node Version:** v6.9.4
* **npm Version:** v4.4.4

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**
(yml)
```
env:
  es6: true
  node: true
extends: 'eslint:recommended'
parserOptions:
  sourceType: module
rules:
  indent:
    - error
    - tab
  quotes:
    - error
    - single
  semi:
    - error
    - always
  object-curly-spacing:
    - error
    - always
  eol-last:
    - error
    - always
  no-multiple-empty-lines:
    - 2
```

**What did you do? Please include the actual source code causing the issue.**
(un-formated js code)
```js
/*eslint no-undef: 0*/

var supertest = require('supertest');
var expect = require('chai').expect;
var pull = require('app-root-path').require;
var app = pull('app.js');

describe('Test for saving order form', function() {

    it('obtain Authorization header by login', function(done) {
    supertest(app).post('/v1/auth/login')
            .send({
         username: 'test123@gmail.com',
                password: '12345678'
    })
            .expect(200)
       .end(function(err, res) {
                if (err) return done(err);
            else {
                    expect(res.body.token).to.have.property('AccessToken');
                    // set token to use in subsequent requests
                    token = res.body.token.AccessToken;
                    done();
                }
            });
    });
});
```
**What did you expect to happen?**
(formatted js code)
```js
/*eslint no-undef: 0*/

var supertest = require('supertest');
var expect = require('chai').expect;
var pull = require('app-root-path').require;
var app = pull('app.js');

describe('Test for saving order form', function() {

    it('obtain Authorization header by login', function(done) {
        supertest(app).post('/v1/auth/login')
            .send({
                username: 'test123@gmail.com',
                password: '12345678'
            })
            .expect(200)
            .end(function(err, res) {
                if (err) return done(err);
                else {
                    expect(res.body.token).to.have.property('AccessToken');
                    // set token to use in subsequent requests
                    token = res.body.token.AccessToken;
                    done();
                }
            });
    });
});
```
**What actually happened? Please include the actual, raw output from ESLint.**

```
➜  lme master ✗ eslint . --fix
```
resulted in
```js
/*eslint no-undef: 0*/

var supertest = require('supertest');
var expect = require('chai').expect;
var pull = require('app-root-path').require;
var app = pull('app.js');

describe('Test for saving order form', function() {

	it('obtain Authorization header by login', function(done) {
		supertest(app).post('/v1/auth/login')
            .send({
	username: 'test123@gmail.com',
	password: '12345678'
})
            .expect(200)
            .end(function(err, res) {
	if (err) return done(err);
	else {
		expect(res.body.token).to.have.property('AccessToken');
                    // set token to use in subsequent requests
		token = res.body.token.AccessToken;
		done();
	}
});
	});
});

```

**Comment by platinumazure** [comment]
Pinging @not-an-aardvark, @kaicataldo, can either of you take a look?

**Comment by not-an-aardvark** [comment]
This was fixed by https://github.com/eslint/eslint/pull/7618. The fix was released in v4.0.0-alpha.0 today, although if you're using the linter in production I would recommend waiting until the main 4.0.0 release.

**Comment by vajahath** [comment]
Alright 👍
